webpack.config.js

```diff
@@ -290,6 +290,7 @@ var config = {
                 {from: 'images/warning.png', to: 'images'},
                 {from: 'images/logo-email.png', to: 'images'},
                 {from: 'images/browser-icons', to: 'images/browser-icons'},
+                {from: 'images/cloud', to: 'images'},
```

Comment:
Should this be limited to the specific png we want?

Reply:
Thinking here was that any future assets could be added without messing with webpack. I don't think there's any significant negative impact, but I'm happy to change to be explicit - let me know what you think!

@devinbinnie (will continue your discussion here) This shouldn't break anything, simply marks it in webpack to be servedby URL /static/images/<png>

Reply:
Okay, as long as there's no explicit reference to the images (eg. in CSS files)

Reply:
Since those are included via imports, they should be fine. I will be sure to give one last check before merging.

Reply:
Confirmed, good to go!